src/main/scala/subsystem/RocketSubsystem.scala

```diff
@@ -20,7 +20,12 @@ case class RocketCrossingParams(
   mmioBaseAddressPrefixWhere: TLBusWrapperLocation = CBUS
 ) extends TileCrossingParamsLike {
   def injectClockNode(context: Attachable)(implicit p: Parameters): ClockNode = {
-    ClockTempNode() // TODO reset stretcher could go here, parameterized by args to this case class
+    if (p(SubsystemResetSchemeKey) != ResetSynchronous) {
+      val rs = LazyModule(new ResetStretcher(16))
```

Comment:
Can this 16 be a parameter?

Reply:
Haha jinx.

While this class is called `RocketCrossingParams` I think it is currently being used for a lot of different types of large-ish Tiles that might have different reset length requirements...

Reply:
Changed to Option[Int] parameter.